src/Components/hero-section/hero-section.styles.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable */
+import styled, { createGlobalStyle } from 'styled-components';
+
+import SVG from '../svg/svg.component';
+import ButtonComponent from '../button/button.component';
+import { ReactComponent as Triangle } from './assets/triangle.svg';
+
+const MonserratCDN =
+  'https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap';
+
+export const GlobalStyles = createGlobalStyle`
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
```

Comment:
Why is this box-sizing given ? it doesnt reflect in the component even though i remove it ? but i affects other component 🤔

Reply:
'box-sizing: border-box' takes into account the margin and padding set to the element when calculating its width.
It makes life a lot easier.